app/code/Magento/Review/Block/Adminhtml/Edit/Form.php

```diff
@@ -75,18 +75,23 @@ protected function _prepareForm()
         $review = $this->_coreRegistry->registry('review_data');
         $product = $this->_productFactory->create()->load($review->getEntityPkValue());
 
+        $formActionParam['id'] = $this->getRequest()->getParam('id');
```

Comment:
`formActionParams` and please do initial assignment like `$var = ['id' => ..., 'ret' => ...]`

Reply:
Hello @orlangur 
I have change `formActionParams` and add initial assignment